mailet/base/src/test/java/org/apache/mailet/base/test/FakeMail.java

```diff
@@ -416,7 +416,6 @@ public final boolean equals(Object o) {
             FakeMail that = (FakeMail) o;
 
             return Objects.equal(this.size, that.size)
-                && Objects.equal(this.msg, that.msg)
```

Comment:
2 (fake) mails with same headers are the same? It seems error prone

Reply:
msg being transiant, EqualsVerifier asks me politely to exclude it from the equals and hashcode.

Note: MimeMessage do probably not implement equals well, we generally rely on other means for testing mail equality cf https://github.com/linagora/james-project/pull/1231#discussion_r161710278 I guess it will not harm...